teku/src/main/java/tech/pegasys/teku/cli/options/ZkChainOptions.java

```diff
@@ -48,12 +49,22 @@ public class ZkChainOptions {
   private int statelessMinProofsRequired =
       ZkChainConfiguration.DEFAULT_STATELESS_MIN_PROOFS_REQUIRED;
 
+  @CommandLine.Option(
+      hidden = true,
+      names = {"--Xstateless-proofs-generation-delay"},
+      paramLabel = "<DURATION>",
+      description = "Proof generation artificial delay in milliseconds.",
+      arity = "1")
+  private long statelessProofGenerationDelay =
+      ZkChainConfiguration.DEFAULT_PROOF_GENERATION_DELAY.toMillis();
```

Comment:
along the same lines, we've got millis here, seconds other places, so being consistent will make it easier to understand.